common/data/src/index.ts

```diff
@@ -20,8 +20,13 @@ export function schemaToGlueColumns(jsonSchema: JsonSchema): glue.Column[] {
     let type: glue.Type = glue.Schema[typeKey]
 
     if (name.endsWith('_at')) type = glue.Schema.DATE
+
     if (name === 'candidate_list') type = glue.Schema.array(glue.Schema.STRING)
 
+    if (name === 'amount') type = glue.Schema.BIG_INT
```

Comment:
Would .endsWith('amount') work well here moving forward?

Similar convention to 'at'.

Reply:
Yes it is much better, thanks